scripts/utils/EnsureAllPackagesExist.ps1

```diff
@@ -27,7 +27,6 @@ Param (
 "NuGet.Localization",
 "NuGet.PackageManagement",
 "NuGet.Packaging.Core",
-"NuGet.Packaging.Extraction",
```

Comment:
Need to check where this script is used. NuGet.Packaging.Extraction should already be excluded from our nuget.org publishing, but I'm worried this script will break our publishing pipeline for servicing branches.

Reply:
Did you find it? :)

Reply:
It looks like we'll be ok. The script passes if `NuGet.Packaging.Extraction` (or other nupkgs) exist and are not defined in the list. In theory it means that if a servicing branch fails to create the N.P.E package, then when we publish it won't detect the missing package. But this package isn't pusblished to nuget.org anyway, only to the daily build feed, which happens as part of the build pipeline, not a release pipeline.